.kokoro/presubmit/presubmit.cfg

```diff
@@ -1,7 +1,7 @@
 # Format: //devtools/kokoro/config/proto/build.proto
 
-# Only run a subset of all nox sessions
+# Disable system tests.
```

Comment:
@vchudnov-g How can we instruct OwlBot to leaver this file as-is, or alternatively where can we update the template for this file? We don't want Kokoro to run all nox sessions, as it takes way too much time. Instead, we have set up GitHub Actions runners (in parallel) for all the nox sessions that we want to run, and only run a couple here.

Reply:
It sounds like this was addressed in https://github.com/googleapis/python-spanner/pull/1373 ?